examples/AllTests/AllTests.cpp

```diff
@@ -31,22 +31,18 @@
 #include "CppUTestExt/IEEE754ExceptionsPlugin.h"
 #include "CppUTestExt/MockSupportPlugin.h"
 
-class MyDummyComparator : public MockNamedValueComparator
-{
+class MyDummyComparator : public MockNamedValueComparator {
 public:
-    virtual bool isEqual(const void* object1, const void* object2)
-    {
+    virtual bool isEqual(const void* object1, const void* object2) {
         return object1 == object2;
     }
 
-    virtual SimpleString valueToString(const void* object)
-    {
+    virtual SimpleString valueToString(const void* object) {
```

Comment:
I think, right now, almost everywhere in CppUTest it has the opening bracket of functions (and TEST macros) on a new line. Personally I do not see a benefit of changing this? Can we configure it so to use that style ? Anything against that?